core/Translation/Translator.php

```diff
@@ -90,6 +104,67 @@ public function translate($translationId, $args = array(), $language = null)
     }
 
     /**
+     * Returns the country name for the given country code translated to the given language
+     *
+     * @param string $countryCode ISO 3166-1 alpha-2 country code
+     * @param null|string $language ISO 639-1 language code   (defaults to current language)
+     * @return string  translated country name
+     */
+    public function getTranslatedCountry($countryCode, $language=null)
+    {
+        $filePattern = PIWIK_INCLUDE_PATH . '/vendor/piwik/i18n-data/countries/%s.json';
```

Comment:
I'm just curious. Is it best practice to compose a bath to `vendor` like this? Should we maybe have it somewhere in the DI container or is there a method that we can use or so? Maybe it is best practice so just ignore :) Maybe @mnapoli knows more?